src/app/shared/components/application-list/style.scss

```diff
@@ -48,7 +48,7 @@ $application-card-width: 320px;
   }
 
   .application-card {
-    @include mixins.size($application-card-width, null, true);
+    @include mixins.size($application-card-width, null);
```

Comment:
See Point in Ticket
- Remove additional spacing between Applications and application cards

![additional-spacing](https://user-images.githubusercontent.com/17727069/201313490-c59720de-1034-4f62-9b81-486aca40f3af.png)

Note:**Since shared component is used so be careful on change**

![applications -list page](https://user-images.githubusercontent.com/17727069/201313605-4543cdf3-1e6f-478f-a046-6957dcad4921.png)

Reply:
I think that point was related to spacing between `Applications` heading and application cards.